app/components/candidate_interface/restructured_work_history/gap_component.html.erb

```diff
@@ -1,6 +1,10 @@
 <%= govuk_inset_text(classes: 'app-inset-text--important govuk-!-margin-top-0 govuk-!-margin-bottom-0') do %>
   You have a break in your work history (<%= format_months_to_years_and_months(@break_period.length) %>)
   <br>
-  <%= govuk_link_to 'Add another job', candidate_interface_new_restructured_work_history_path %> or
-  <%= govuk_link_to 'add a reason for this break', candidate_interface_new_restructured_work_history_break_path(start_date: @break_period.start_date, end_date: @break_period.end_date) %>
+  <%= govuk_link_to candidate_interface_new_restructured_work_history_path do %>
+    Add another job <span class="govuk-visually-hidden"><%= between_formatted_dates %></span>
```

Comment:
This should hopefully fix this whitespace issue with the underline:

<img width="440" alt="Screenshot 2021-04-28 at 18 55 32" src="https://user-images.githubusercontent.com/813383/116450559-5bc01900-a853-11eb-8e6a-fbf68cd7aec9.png">

```suggestion
    Add another job<span class="govuk-visually-hidden"> <%= between_formatted_dates %></span>
```

Reply:
Okay so a bit of a weird one... i made the changes you suggested and the whitespece remained there 🤔 

It seems that it's being rendered after the span but only for the first link.

After playing around for a bit I managed to get rid of it by pulling the <% end %> up onto the same line. 

The front end looks right but the html.erb markup is a bit ugly, is this okay? 

Happy to go with a different solution if you can think of one :)